### PR TITLE
1ES Migration

### PR DESCRIPTION
**Pull request by danieljurek**
Migrates to 1ES Pipeline Templates: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/overview

The following pipelines are migrated: 
* azure-dev - cli 
* azure-dev - vscode
* azure-dev - repoman

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]

File: eng/pipelines/repoman.yml [hunk 1]
```diff
@@ -187,4 +156,52 @@ stages:
...
+        variables:
+          - template: /eng/pipelines/templates/variables/globals.yml
+
+        jobs:
+        - deployment: Publish_Repos
+          environment: azure-dev
+
+          pool:
+            name: azsdk-pool-mms-ubuntu-2004-general
+            image: azsdk-pool-mms-ubuntu-2004-1espt
```
Comment (benbp) [inline eng/pipelines/repoman.yml:183]: Don't have to worry about it here, but a follow-on to use `image.yml` would be nice.

File: eng/pipelines/templates/stages/publish.yml [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+        - deployment: Publish_Release
+          condition: >-
+            and(
+              succeeded(),
+              ne('true', variables['Skip.Publish'])
+            )
+          environment: azure-dev
+
+          pool:
+            name: $(LINUXPOOL)
```
Comment (benbp) [inline eng/pipelines/templates/stages/publish.yml:32]: Does this work? I could not get pool vars working in deployment contexts.

File: templates/common/NOTICE.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
+Test change
```
Comment (benbp) [inline templates/common/NOTICE.txt:1]:
```suggestion
```

**Review by benbp — APPROVED** [review]
Some questions/edits otherwise LGTM

File: eng/pipelines/repoman.yml (hunk 1 above)
Comment (danieljurek) [reply eng/pipelines/repoman.yml:183]: This is a `deployment`

File: eng/pipelines/templates/stages/publish.yml (hunk 2 above)
Comment (danieljurek) [reply eng/pipelines/templates/stages/publish.yml:32]: Probably not. Exercising releases requires a lot of disabling and commenting and isn't always a proper test. I'll update and give it a cursory run.

**Pull request merged**